winter_messaging_outbox_transacton/start_worker.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import importlib
+import logging
+import sys
+
+from injector import Injector
+
+from winter_messaging_outbox_transacton.consumer_worker import ConsumerWorker
+from winter_messaging_outbox_transacton.injection import Configuration
+from winter_messaging_outbox_transacton.middleware_registry import MiddlewareRegistry
+from winter_messaging_outbox_transacton.worker_configuration import WorkerConfiguration
+
+logger = logging.getLogger('event_handling')
+
+
+class Parser(argparse.ArgumentParser):
+    def error(self, message):
+        sys.stderr.write('error: %s\n' % message)
+        self.print_help()
+        sys.exit(2)
+
+
+def parse_args(parser_object):
+    parser_object.add_argument(
+        'consumer',
+        type=str,
+        help='Consumer ID',
+    )
+    parser_object.add_argument(
+        '-p',
+        '--package-module',
+        type=str,
+        dest='package_module',
+        help='',
+    )
+    return parser_object.parse_args()
+
+
+if __name__ == '__main__':
+    import django
```

Comment:
why do you have import here?

there is no funcitons which can be used outside
so it seems that this file should not be imported and you can import everything at the begging of the file